### PR TITLE
Update online_accounts_to_join.md

### PR DESCRIPTION
**Pull request by fcerpe**
- copied text
- expanded mailing list and other accounts
- added links, as much as possible

### CODE REVIEW

File: doc/online_accounts_to_join.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Online accounts to join
+Here is an overview of means of communications that you should be part of. 
+
+
+## Mailing lists
+Normally, you are automatically added to all the following lists, or it's a human process that can require a few days. 
+If that is not the case and you hear lab members talk about events you are not aware of, here's a list of mailing lists and who to contact to be added
+
+#### CPP_LLN mailing list
```
Comment (marcobarilari) [inline doc/online_accounts_to_join.md:9]:
```suggestion
### CPP_LLN mailing list
```

File: doc/online_accounts_to_join.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+- for anything lab-related
+- contact any CPP member 
+
+#### Communaute-ipsy
```
Comment (marcobarilari) [inline doc/online_accounts_to_join.md:13]:
```suggestion
### Communaute-ipsy
```

File: doc/online_accounts_to_join.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+- to be informed about the seminars / talks / events in our department in LLN
+- contact Nadine Fraselle
+
+#### Neurocs-Ipsy (deprected)
```
Comment (marcobarilari) [inline doc/online_accounts_to_join.md:17]:
```suggestion
### Neurocs-Ipsy (deprected)
```

File: doc/online_accounts_to_join.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+#### IoNS members
```
Comment (marcobarilari) [inline doc/online_accounts_to_join.md:19]:
```suggestion
### IoNS members
```

File: doc/online_accounts_to_join.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+- to be in the loop of events in IoNS (Institute of Neuroscience in Woluwe)
+- contact Cathy Friand
+
+### Setup Gmail labels
```
Comment (marcobarilari) [inline doc/online_accounts_to_join.md:23]:
```suggestion
[!TIP] Setup Gmail labels
```

File: doc/online_accounts_to_join.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
+# Online accounts to join
+Here is an overview of means of communications that you should be part of. 
+
+
+## Mailing lists
+Normally, you are automatically added to all the following lists, or it's a human process that can require a few days. 
+If that is not the case and you hear lab members talk about events you are not aware of, here's a list of mailing lists and who to contact to be added
+
+#### CPP_LLN mailing list
+- for anything lab-related
```
Comment (marcobarilari) [inline doc/online_accounts_to_join.md:10]: I would define the "anything" and say that the email list is a google group

**Review by marcobarilari — CHANGES_REQUESTED** [review]

**Comment by marcobarilari** [comment]
Thank you for taking care of this, LMK if something is not clear

File: doc/online_accounts_to_join.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+- contact Nadine Fraselle
+
+#### Neurocs-Ipsy (deprected)
+
+#### IoNS members
+- to be in the loop of events in IoNS (Institute of Neuroscience in Woluwe)
+- contact Cathy Friand
+
+### Setup Gmail labels
+If you want to have emails from the CPP_LLN mailing list clearly marked in your PRIMARY tab on GMail (and potentially other email platforms too), follow these simple steps:
```
Comment (marcobarilari) [inline doc/online_accounts_to_join.md:24]:
```suggestion
**Setup Gmail labels**

If you want to have emails from the CPP_LLN mailing list clearly marked in your PRIMARY tab on GMail (and potentially other email platforms too), follow these simple steps: 
```

Comment (marcobarilari) [reply doc/online_accounts_to_join.md:26]: Ok, I need to edit this locally after you have accepted this suggestion

File: doc/online_accounts_to_join.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
+# Online accounts to join
+Here is an overview of means of communications that you should be part of. 
+
+
+## Mailing lists
+Normally, you are automatically added to all the following lists, or it's a human process that can require a few days. 
+If that is not the case and you hear lab members talk about events you are not aware of, here's a list of mailing lists and who to contact to be added
+
+### CPP_LLN mailing list
+- for anything lab-related
```
Comment (fcerpe) [inline doc/online_accounts_to_join.md:10]:
```suggestion
- for lab-related information: lab meetings, grants, interesting talks and events, general communications. This email list is a google group, meaning that a message sent to this account will be sent to anyone in the list 
```

**Review by fcerpe — COMMENTED** [review]
Indentations are implemented
Lab mailing list is more detailed

**Review by marcobarilari — APPROVED** [review]

**Pull request merged**